packages/gafl-webapp-service/src/services/payment/__test__/govuk-pay-service.spec.js

```diff
@@ -21,7 +21,7 @@ describe('The govuk-pay-service', () => {
     ).toEqual({
       amount: 5400,
       delayed_capture: false,
-      description: 'Senior, Salmon and sea trout, 12 months',
+      description: 'Over 65, Salmon and sea trout, 12 months',
```

Comment:
Would be nice to use the SENIOR_MIN_AGE constant from business rules lib for this.

Reply:
It uses the concessions which already use the business rules library that so its circular. This is the for the display of the product it's a to generate the product description throughout. I think bringing in the age variable again here is pointless.